dahu/core/app/scripts/views/workspace/objects/tooltip.js

```diff
@@ -34,6 +35,10 @@ define([
 
         className: 'object tooltip ui-widget-content',
 
+        initialize: function() {
+            this.changeBackgroundColor();
```

Comment:
But when someone will change the default color in the model, he will have to know that he has to change the css. I still do it ?